api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -1407,22 +1407,23 @@ public void retireOrderFrequency_shouldRetireGivenOrderFrequency() throws Except
 	@Test
 	@Verifies(value = "should unretire given order frequency", method = "unretireOrderFrequency(OrderFrequency)")
 	public void unretireOrderFrequency_shouldUnretireGivenOrderFrequency() throws Exception {
-		OrderFrequency orderFrequency = orderService.getOrderFrequency(3);
+		executeDataSet("org/openmrs/api/include/OrderServiceTest-otherOrderFrequencies.xml");
+		OrderFrequency orderFrequency = orderService.getOrderFrequency(103);
 		assertNotNull(orderFrequency);
 		assertTrue(orderFrequency.getRetired());
-		assertEquals("Some Retire Reason", orderFrequency.getRetireReason());
```

Comment:
@dkayiwa yes. the dataset has been replaced and this change is for the new dataset since its retire reason is null.